### ISSUE TITLE
Bypassing popup annoys me asf

### ISSUE THREAD

**Issue by kartavkun** [YTLitePlus/YTLitePlus#470]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

❌ No, I don't think this is a bug. I will explain below

### My question

I’ve searched for this issue, but no success. I really want to know can I turn off this feature?

https://github.com/user-attachments/assets/3e35f043-a5eb-426a-ae9b-605545d3c57b

**Comment by kartavkun** [comment]
I installed YTLitePlus from AltStore source

**Comment by Taftle10** [comment]
The Altstore source has not been updated yet, you can make your own build of YTLitePlus by following the wiki. If you do not know how to find the wiki like me just go here: **https://github.com/YTLitePlus/YTLitePlus** And look at the top bar with all of the _Code, Issues, etc._ And click the wiki tab, it will tell you how to make your own YTLitePlus IPA. Now, when I built it on my own I had to edit the code slightly by simply adding the code related to "Bypassing Popup" and simply deleting the code on line 179, which is the part that actually shows the notification/toast. Otherwise, the popup that the code blocks will appear which is equally annoying. You will need a decrypted IPA of YouTube (I'm not sure if I am allowed to link to the website with it directly but I found that _Armconverter_ had the IPA and it worked fine for me. Sorry if this is hard to follow as I am new to this stuff in general and am not sure how to describe it well.
